### DISCUSSION TITLE
Does Firefly not support CLI when using remote user authentication?

### DISCUSSION THREAD

**Discussion by nabeelmoeen** [firefly-iii/firefly-iii#8053] [Firefly III]
I keep on getting the 'Invalid token' error when using the personal access token to run CLI commands.
I am using remote User authentication.

Is the CLI not supported when user the remote user authentication?

Firefly also does not generate a new personal access token whten using this option

**Comment by JC5** [comment]
You need the command line token on the /profile page. Second tab.

**Reply by nabeelmoeen** [reply]
Thanks James, 
Yes that's what I am using:
![Screenshot 2023-10-12 161739](https://github.com/firefly-iii/firefly-iii/assets/6676242/6155f365-1d4c-40ca-a3cc-03073ec517f0)

but still getting the error:
![Screenshot 2023-10-12 162320](https://github.com/firefly-iii/firefly-iii/assets/6676242/754cf7bb-4a38-402f-a872-2670b0dddd55)


My question was to confirm if the CLI will continue to work with the Command line token normally, if the application has been set to use Remote User authentication.

**Reply by JC5** [reply]
Yes. Are you sure you're user 1?

**Reply by nabeelmoeen** [reply]
I think I just set up one user..is there a way to tell?

**Reply by JC5** [reply]
Your `/profile` page should mention it on the first tab.

**Reply by nabeelmoeen** [reply]
awesome, thanks a lot. yes, as it turns out my user id was 2.

appreciate the quick response.
